packages/ts-transform/src/utils/ast-node.tsx

```diff
@@ -21,6 +21,8 @@ export const getIdentifierText = (
   return ((node as ts.Identifier).escapedText as string) || (node as ts.Identifier).text;
 };
 
+export const getDeclarationValue = (node: ts.VariableDeclaration) => node.initializer?.getText();
```

Comment:
if we want it without the quotes we can do (code wont be 100% correct):

```js
export const getDeclarationValue = (node: ts.VariableDeclaration) => {
  if (ts.isLiteral(node.initializer) {
    return node.initializer.text;
  }
  
  throw createNodeError(node, 'should have been a literal node');
};
```

Reply:
Thanks so much, I spent way too much time understanding the nodes and types.